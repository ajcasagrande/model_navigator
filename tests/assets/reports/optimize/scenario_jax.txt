OptimizeEvent.OPTIMIZATION_STARTED {}
OptimizeEvent.PIPELINE_STARTED {'name': 'Preprocessing'}
OptimizeEvent.COMMAND_STARTED {'command': 'LoadMetadata', 'config_key': None, 'runner_name': None}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
OptimizeEvent.PIPELINE_FINISHED {}
OptimizeEvent.PIPELINE_STARTED {'name': 'TensorFlow 2 Conversion'}
OptimizeEvent.COMMAND_STARTED {'command': 'ConvertSavedModel2ONNX', 'config_key': 'onnx-jit-xla', 'runner_name': None}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.FAIL}
OptimizeEvent.COMMAND_STARTED {'command': 'GraphSurgeonOptimize', 'config_key': 'onnx-jit-xla', 'runner_name': None}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'ConvertSavedModel2ONNX', 'config_key': 'onnx-jit', 'runner_name': None}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.FAIL}
OptimizeEvent.COMMAND_STARTED {'command': 'GraphSurgeonOptimize', 'config_key': 'onnx-jit', 'runner_name': None}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'ConvertSavedModel2ONNX', 'config_key': 'onnx-xla', 'runner_name': None}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.FAIL}
OptimizeEvent.COMMAND_STARTED {'command': 'GraphSurgeonOptimize', 'config_key': 'onnx-xla', 'runner_name': None}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'ConvertSavedModel2ONNX', 'config_key': 'onnx', 'runner_name': None}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.FAIL}
OptimizeEvent.COMMAND_STARTED {'command': 'GraphSurgeonOptimize', 'config_key': 'onnx', 'runner_name': None}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.PIPELINE_FINISHED {}
OptimizeEvent.PIPELINE_STARTED {'name': 'TensorRT Conversion'}
OptimizeEvent.COMMAND_STARTED {'command': 'ConvertONNX2TRT', 'config_key': 'trt-jit-xla-fp32', 'runner_name': None}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'ConvertONNX2TRT', 'config_key': 'trt-jit-xla-fp16', 'runner_name': None}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'ConvertONNX2TRT', 'config_key': 'trt-jit-fp32', 'runner_name': None}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'ConvertONNX2TRT', 'config_key': 'trt-jit-fp16', 'runner_name': None}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'ConvertONNX2TRT', 'config_key': 'trt-xla-fp32', 'runner_name': None}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'ConvertONNX2TRT', 'config_key': 'trt-xla-fp16', 'runner_name': None}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'ConvertONNX2TRT', 'config_key': 'trt-fp32', 'runner_name': None}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'ConvertONNX2TRT', 'config_key': 'trt-fp16', 'runner_name': None}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.PIPELINE_FINISHED {}
OptimizeEvent.PIPELINE_STARTED {'name': 'Correctness'}
OptimizeEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'tf-savedmodel-jit-xla', 'runner_name': 'TensorFlowSavedModelCUDA'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
OptimizeEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'tf-savedmodel-jit', 'runner_name': 'TensorFlowSavedModelCUDA'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
OptimizeEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'tf-savedmodel-xla', 'runner_name': 'TensorFlowSavedModelCUDA'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
OptimizeEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'tf-savedmodel', 'runner_name': 'TensorFlowSavedModelCUDA'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
OptimizeEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'onnx-jit-xla', 'runner_name': 'OnnxCUDA'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'onnx-jit-xla', 'runner_name': 'OnnxTensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'onnx-jit', 'runner_name': 'OnnxCUDA'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'onnx-jit', 'runner_name': 'OnnxTensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'onnx-xla', 'runner_name': 'OnnxCUDA'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'onnx-xla', 'runner_name': 'OnnxTensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'onnx', 'runner_name': 'OnnxCUDA'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'onnx', 'runner_name': 'OnnxTensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'trt-jit-xla-fp32', 'runner_name': 'TensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'trt-jit-xla-fp16', 'runner_name': 'TensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'trt-jit-fp32', 'runner_name': 'TensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'trt-jit-fp16', 'runner_name': 'TensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'trt-xla-fp32', 'runner_name': 'TensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'trt-xla-fp16', 'runner_name': 'TensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'trt-fp32', 'runner_name': 'TensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.FAIL}
OptimizeEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'trt-fp16', 'runner_name': 'TensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.PIPELINE_FINISHED {}
OptimizeEvent.PIPELINE_STARTED {'name': 'Performance'}
OptimizeEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'tf-savedmodel-jit-xla', 'runner_name': 'TensorFlowSavedModelCUDA'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
OptimizeEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'tf-savedmodel-jit', 'runner_name': 'TensorFlowSavedModelCUDA'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
OptimizeEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'tf-savedmodel-xla', 'runner_name': 'TensorFlowSavedModelCUDA'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
OptimizeEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'tf-savedmodel', 'runner_name': 'TensorFlowSavedModelCUDA'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
OptimizeEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'onnx-jit-xla', 'runner_name': 'OnnxCUDA'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'onnx-jit-xla', 'runner_name': 'OnnxTensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'onnx-jit', 'runner_name': 'OnnxCUDA'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'onnx-jit', 'runner_name': 'OnnxTensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'onnx-xla', 'runner_name': 'OnnxCUDA'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'onnx-xla', 'runner_name': 'OnnxTensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'onnx', 'runner_name': 'OnnxCUDA'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'onnx', 'runner_name': 'OnnxTensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'trt-jit-xla-fp32', 'runner_name': 'TensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'trt-jit-xla-fp16', 'runner_name': 'TensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'trt-jit-fp32', 'runner_name': 'TensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'trt-jit-fp16', 'runner_name': 'TensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'trt-xla-fp32', 'runner_name': 'TensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'trt-xla-fp16', 'runner_name': 'TensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'trt-fp32', 'runner_name': 'TensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.FAIL}
OptimizeEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'trt-fp16', 'runner_name': 'TensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.PIPELINE_FINISHED {}
OptimizeEvent.PIPELINE_STARTED {'name': 'Verify Models'}
OptimizeEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'tf-savedmodel-jit-xla', 'runner_name': 'TensorFlowSavedModelCUDA'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'tf-savedmodel-jit', 'runner_name': 'TensorFlowSavedModelCUDA'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'tf-savedmodel-xla', 'runner_name': 'TensorFlowSavedModelCUDA'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'tf-savedmodel', 'runner_name': 'TensorFlowSavedModelCUDA'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'onnx-jit-xla', 'runner_name': 'OnnxCUDA'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'onnx-jit-xla', 'runner_name': 'OnnxTensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'onnx-jit', 'runner_name': 'OnnxCUDA'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'onnx-jit', 'runner_name': 'OnnxTensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'onnx-xla', 'runner_name': 'OnnxCUDA'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'onnx-xla', 'runner_name': 'OnnxTensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'onnx', 'runner_name': 'OnnxCUDA'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'onnx', 'runner_name': 'OnnxTensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'trt-jit-xla-fp32', 'runner_name': 'TensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'trt-jit-xla-fp16', 'runner_name': 'TensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'trt-jit-fp32', 'runner_name': 'TensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'trt-jit-fp16', 'runner_name': 'TensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'trt-xla-fp32', 'runner_name': 'TensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'trt-xla-fp16', 'runner_name': 'TensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'trt-fp32', 'runner_name': 'TensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'trt-fp16', 'runner_name': 'TensorRT'}
OptimizeEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
OptimizeEvent.PIPELINE_FINISHED {}
OptimizeEvent.BEST_MODEL_PICKED {'config_key': 'tf-savedmodel', 'runner_name': 'TensorFlowSavedModelCUDA', 'model_path': PosixPath('/workspace/navigator_workspace/tf-savedmodel/model.savedmodel')}
OptimizeEvent.OPTIMIZATION_FINISHED {}
