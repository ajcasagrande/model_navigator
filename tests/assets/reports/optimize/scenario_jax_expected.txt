Optimization started.
Model: Collecting model information ...
Model: Collecting model information OK
Model: Building ONNX model from TensorFlow SavedModel ...
Model: Building ONNX model from TensorFlow SavedModel FAIL
Model: Optimizing graph for onnx-jit-xla ...
Model: Optimizing graph for onnx-jit-xla SKIPPED
Model: Building ONNX model from TensorFlow SavedModel ...
Model: Building ONNX model from TensorFlow SavedModel FAIL
Model: Optimizing graph for onnx-jit ...
Model: Optimizing graph for onnx-jit SKIPPED
Model: Building ONNX model from TensorFlow SavedModel ...
Model: Building ONNX model from TensorFlow SavedModel FAIL
Model: Optimizing graph for onnx-xla ...
Model: Optimizing graph for onnx-xla SKIPPED
Model: Building ONNX model from TensorFlow SavedModel ...
Model: Building ONNX model from TensorFlow SavedModel FAIL
Model: Optimizing graph for onnx ...
Model: Optimizing graph for onnx SKIPPED
Model: Building TensorRT engine trt-jit-xla-fp32 from ONNX model ...
Model: Building TensorRT engine trt-jit-xla-fp32 from ONNX model SKIPPED
Model: Building TensorRT engine trt-jit-xla-fp16 from ONNX model ...
Model: Building TensorRT engine trt-jit-xla-fp16 from ONNX model SKIPPED
Model: Building TensorRT engine trt-jit-fp32 from ONNX model ...
Model: Building TensorRT engine trt-jit-fp32 from ONNX model SKIPPED
Model: Building TensorRT engine trt-jit-fp16 from ONNX model ...
Model: Building TensorRT engine trt-jit-fp16 from ONNX model SKIPPED
Model: Building TensorRT engine trt-xla-fp32 from ONNX model ...
Model: Building TensorRT engine trt-xla-fp32 from ONNX model SKIPPED
Model: Building TensorRT engine trt-xla-fp16 from ONNX model ...
Model: Building TensorRT engine trt-xla-fp16 from ONNX model SKIPPED
Model: Building TensorRT engine trt-fp32 from ONNX model ...
Model: Building TensorRT engine trt-fp32 from ONNX model SKIPPED
Model: Building TensorRT engine trt-fp16 from ONNX model ...
Model: Building TensorRT engine trt-fp16 from ONNX model SKIPPED
Model: Validating model tf-savedmodel-jit-xla on TensorFlowSavedModelCUDA backend ...
Model: Validating model tf-savedmodel-jit-xla on TensorFlowSavedModelCUDA backend OK
Model: Validating model tf-savedmodel-jit on TensorFlowSavedModelCUDA backend ...
Model: Validating model tf-savedmodel-jit on TensorFlowSavedModelCUDA backend OK
Model: Validating model tf-savedmodel-xla on TensorFlowSavedModelCUDA backend ...
Model: Validating model tf-savedmodel-xla on TensorFlowSavedModelCUDA backend OK
Model: Validating model tf-savedmodel on TensorFlowSavedModelCUDA backend ...
Model: Validating model tf-savedmodel on TensorFlowSavedModelCUDA backend OK
Model: Validating model onnx-jit-xla on OnnxCUDA backend ...
Model: Validating model onnx-jit-xla on OnnxCUDA backend SKIPPED
Model: Validating model onnx-jit-xla on OnnxTensorRT backend ...
Model: Validating model onnx-jit-xla on OnnxTensorRT backend SKIPPED
Model: Validating model onnx-jit on OnnxCUDA backend ...
Model: Validating model onnx-jit on OnnxCUDA backend SKIPPED
Model: Validating model onnx-jit on OnnxTensorRT backend ...
Model: Validating model onnx-jit on OnnxTensorRT backend SKIPPED
Model: Validating model onnx-xla on OnnxCUDA backend ...
Model: Validating model onnx-xla on OnnxCUDA backend SKIPPED
Model: Validating model onnx-xla on OnnxTensorRT backend ...
Model: Validating model onnx-xla on OnnxTensorRT backend SKIPPED
Model: Validating model onnx on OnnxCUDA backend ...
Model: Validating model onnx on OnnxCUDA backend SKIPPED
Model: Validating model onnx on OnnxTensorRT backend ...
Model: Validating model onnx on OnnxTensorRT backend SKIPPED
Model: Validating model trt-jit-xla-fp32 on TensorRT backend ...
Model: Validating model trt-jit-xla-fp32 on TensorRT backend SKIPPED
Model: Validating model trt-jit-xla-fp16 on TensorRT backend ...
Model: Validating model trt-jit-xla-fp16 on TensorRT backend SKIPPED
Model: Validating model trt-jit-fp32 on TensorRT backend ...
Model: Validating model trt-jit-fp32 on TensorRT backend SKIPPED
Model: Validating model trt-jit-fp16 on TensorRT backend ...
Model: Validating model trt-jit-fp16 on TensorRT backend SKIPPED
Model: Validating model trt-xla-fp32 on TensorRT backend ...
Model: Validating model trt-xla-fp32 on TensorRT backend SKIPPED
Model: Validating model trt-xla-fp16 on TensorRT backend ...
Model: Validating model trt-xla-fp16 on TensorRT backend SKIPPED
Model: Validating model trt-fp32 on TensorRT backend ...
Model: Validating model trt-fp32 on TensorRT backend FAIL
Model: Validating model trt-fp16 on TensorRT backend ...
Model: Validating model trt-fp16 on TensorRT backend SKIPPED
Model: Benchmarking model tf-savedmodel-jit-xla on TensorFlowSavedModelCUDA backend ...
Model: Benchmarking model tf-savedmodel-jit-xla on TensorFlowSavedModelCUDA backend OK
Model: Benchmarking model tf-savedmodel-jit on TensorFlowSavedModelCUDA backend ...
Model: Benchmarking model tf-savedmodel-jit on TensorFlowSavedModelCUDA backend OK
Model: Benchmarking model tf-savedmodel-xla on TensorFlowSavedModelCUDA backend ...
Model: Benchmarking model tf-savedmodel-xla on TensorFlowSavedModelCUDA backend OK
Model: Benchmarking model tf-savedmodel on TensorFlowSavedModelCUDA backend ...
Model: Benchmarking model tf-savedmodel on TensorFlowSavedModelCUDA backend OK
Model: Benchmarking model onnx-jit-xla on OnnxCUDA backend ...
Model: Benchmarking model onnx-jit-xla on OnnxCUDA backend SKIPPED
Model: Benchmarking model onnx-jit-xla on OnnxTensorRT backend ...
Model: Benchmarking model onnx-jit-xla on OnnxTensorRT backend SKIPPED
Model: Benchmarking model onnx-jit on OnnxCUDA backend ...
Model: Benchmarking model onnx-jit on OnnxCUDA backend SKIPPED
Model: Benchmarking model onnx-jit on OnnxTensorRT backend ...
Model: Benchmarking model onnx-jit on OnnxTensorRT backend SKIPPED
Model: Benchmarking model onnx-xla on OnnxCUDA backend ...
Model: Benchmarking model onnx-xla on OnnxCUDA backend SKIPPED
Model: Benchmarking model onnx-xla on OnnxTensorRT backend ...
Model: Benchmarking model onnx-xla on OnnxTensorRT backend SKIPPED
Model: Benchmarking model onnx on OnnxCUDA backend ...
Model: Benchmarking model onnx on OnnxCUDA backend SKIPPED
Model: Benchmarking model onnx on OnnxTensorRT backend ...
Model: Benchmarking model onnx on OnnxTensorRT backend SKIPPED
Model: Benchmarking model trt-jit-xla-fp32 on TensorRT backend ...
Model: Benchmarking model trt-jit-xla-fp32 on TensorRT backend SKIPPED
Model: Benchmarking model trt-jit-xla-fp16 on TensorRT backend ...
Model: Benchmarking model trt-jit-xla-fp16 on TensorRT backend SKIPPED
Model: Benchmarking model trt-jit-fp32 on TensorRT backend ...
Model: Benchmarking model trt-jit-fp32 on TensorRT backend SKIPPED
Model: Benchmarking model trt-jit-fp16 on TensorRT backend ...
Model: Benchmarking model trt-jit-fp16 on TensorRT backend SKIPPED
Model: Benchmarking model trt-xla-fp32 on TensorRT backend ...
Model: Benchmarking model trt-xla-fp32 on TensorRT backend SKIPPED
Model: Benchmarking model trt-xla-fp16 on TensorRT backend ...
Model: Benchmarking model trt-xla-fp16 on TensorRT backend SKIPPED
Model: Benchmarking model trt-fp32 on TensorRT backend ...
Model: Benchmarking model trt-fp32 on TensorRT backend FAIL
Model: Benchmarking model trt-fp16 on TensorRT backend ...
Model: Benchmarking model trt-fp16 on TensorRT backend SKIPPED
Model: Verifying model tf-savedmodel-jit-xla on TensorFlowSavedModelCUDA backend ...
Model: Verifying model tf-savedmodel-jit-xla on TensorFlowSavedModelCUDA backend SKIPPED
Model: Verifying model tf-savedmodel-jit on TensorFlowSavedModelCUDA backend ...
Model: Verifying model tf-savedmodel-jit on TensorFlowSavedModelCUDA backend SKIPPED
Model: Verifying model tf-savedmodel-xla on TensorFlowSavedModelCUDA backend ...
Model: Verifying model tf-savedmodel-xla on TensorFlowSavedModelCUDA backend SKIPPED
Model: Verifying model tf-savedmodel on TensorFlowSavedModelCUDA backend ...
Model: Verifying model tf-savedmodel on TensorFlowSavedModelCUDA backend SKIPPED
Model: Verifying model onnx-jit-xla on OnnxCUDA backend ...
Model: Verifying model onnx-jit-xla on OnnxCUDA backend SKIPPED
Model: Verifying model onnx-jit-xla on OnnxTensorRT backend ...
Model: Verifying model onnx-jit-xla on OnnxTensorRT backend SKIPPED
Model: Verifying model onnx-jit on OnnxCUDA backend ...
Model: Verifying model onnx-jit on OnnxCUDA backend SKIPPED
Model: Verifying model onnx-jit on OnnxTensorRT backend ...
Model: Verifying model onnx-jit on OnnxTensorRT backend SKIPPED
Model: Verifying model onnx-xla on OnnxCUDA backend ...
Model: Verifying model onnx-xla on OnnxCUDA backend SKIPPED
Model: Verifying model onnx-xla on OnnxTensorRT backend ...
Model: Verifying model onnx-xla on OnnxTensorRT backend SKIPPED
Model: Verifying model onnx on OnnxCUDA backend ...
Model: Verifying model onnx on OnnxCUDA backend SKIPPED
Model: Verifying model onnx on OnnxTensorRT backend ...
Model: Verifying model onnx on OnnxTensorRT backend SKIPPED
Model: Verifying model trt-jit-xla-fp32 on TensorRT backend ...
Model: Verifying model trt-jit-xla-fp32 on TensorRT backend SKIPPED
Model: Verifying model trt-jit-xla-fp16 on TensorRT backend ...
Model: Verifying model trt-jit-xla-fp16 on TensorRT backend SKIPPED
Model: Verifying model trt-jit-fp32 on TensorRT backend ...
Model: Verifying model trt-jit-fp32 on TensorRT backend SKIPPED
Model: Verifying model trt-jit-fp16 on TensorRT backend ...
Model: Verifying model trt-jit-fp16 on TensorRT backend SKIPPED
Model: Verifying model trt-xla-fp32 on TensorRT backend ...
Model: Verifying model trt-xla-fp32 on TensorRT backend SKIPPED
Model: Verifying model trt-xla-fp16 on TensorRT backend ...
Model: Verifying model trt-xla-fp16 on TensorRT backend SKIPPED
Model: Verifying model trt-fp32 on TensorRT backend ...
Model: Verifying model trt-fp32 on TensorRT backend SKIPPED
Model: Verifying model trt-fp16 on TensorRT backend ...
Model: Verifying model trt-fp16 on TensorRT backend SKIPPED
Optimization finished for the model.
                           Optimization result for max throughput and min latency strategy                           
┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃ Optimized backend                                 ┃ Path                                                          ┃
┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
│ tf-savedmodel on TensorFlowSavedModelCUDA backend │ /workspace/navigator_workspace/tf-savedmodel/model.savedmodel │
└───────────────────────────────────────────────────┴───────────────────────────────────────────────────────────────┘
